printmyblog.php

```diff
@@ -1,24 +1,25 @@
 <?php
+
 /**
  * @package PrintMyBlog
  * @version 1.0
+ *
+ * @wordpress-plugin
```

Comment:
I'm unfamiliar with this tag. Can you point me to some documentation so I know what this is doing?

Reply:
Here you go https://developer.wordpress.org/plugins/plugin-basics/header-requirements/#header-fields